specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -84,6 +84,101 @@
           }
         }
       }
+    },
+    "/analyze-conversations/jobs": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "description": "Submit a collection of conversations for analysis. Specify one or more unique tasks to be executed.",
+        "operationId": "AnalyzeConversation_SubmitJob",
+        "summary": "Submit analysis job for conversations",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "The collection of conversations to analyze and one or more tasks to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeConversationJobsInput"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Conversational PII Analysis Job Request": {
+            "$ref": "./examples/conversations/SuccessfulConversationPIISubmit.json"
+          },
+          "Successful Conversation Summarization Analysis Job Request": {
+            "$ref": "./examples/conversations/SuccessfulConversationSummarySubmit.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/analyze-conversations/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "description": "Get the status of an analysis job. A job may consist of one or more tasks. Once all tasks are succeeded, the job will transition to the succeeded state and results will be available for each task.",
+        "operationId": "AnalyzeConversation_JobStatus",
+        "summary": "Get analysis status and results",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/JobId"
+          },
+          {
+            "$ref": "common.json#/parameters/ShowStats"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Analysis job status and metadata.",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeConversationJobState"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
```

Comment:
Nit: good to be consistent with the default error description. This differs from above.

Reply:
Changed to             "description": "Error response.",